spec/jobs/get_usps_proofing_results_job_spec.rb

```diff
@@ -1498,24 +1498,28 @@
       end
```

Comment:
(I'm commenting on a file so that we can thread responses to this comment and resolve the conversation when we're done chatting.)

I noticed that the testing plan recommends using `UspsInPersonProofing::Mock::Fixtures.request_passed_proofing_secondary_id_type_results_response`.  When I look at this mock file, it has `"ippAssuranceLevel": "1.5"`.  Since this is EIPP, I think we'd want to use a file with `"ippAssuranceLevel": "2.0"`.

I don't know that this would affect behavior for the testing plan, but I wanted to flag it because that line jumped out at me.

Reply:
I would recommend using fixtures that have the correct assurance level in the specs.  Thoughts?